reference/forms/types/options/label_format.rst.inc

```diff
@@ -0,0 +1,38 @@
+label_format
+~~~~~~~~~~~~
+
+.. versionadded:: 2.6
+    The ``label_format`` option was introduced in Symfony 2.6.
+
+**type**: ``string`` **default**: ``null``
+
+Sets the format used to generate the ``<label>`` element. This is usefull when
```

Comment:
Funny, as I took the first sentence from your PR :wink: However, I agree with you. I've tried to improve it.